### PR TITLE
Zenspider mbertheau fix meetup

### PR DESCRIPTION
**Pull request by jgilray**
This PR is an effort to fix the issues with https://github.com/exercism/racket/pull/56.  I added ```README.md```, the template file ```meetup.rkt``` and reformatted ```config.json```

### CODE REVIEW

File: exercises/meetup/example.rkt [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            [(Thursday) 4]
+            [(Friday) 5]
+            [(Saturday) 6]
+            [else (raise-argument-error 'meetup-day "weekday" 3 year month weekday week)])])
+    (make-date year month
+               (+ first-of-week
+                    (modulo
+                      (- weekday
+                       (date-week-day (make-date year month first-of-week))) 7)
+                 ))))
```
Comment (yurrriq) [inline exercises/meetup/example.rkt]: We should gather trailing parentheses.

**Comment by yurrriq** [comment]
Looks like Travis is upset..

**Comment by jgilray** [comment]
Hi @yurrriq, yow! this may be beyond me.  Clicking on "Details" above it looks like maybe ```.travis.yml``` is really old?  I don't know what that script should do, but it looks very different than the version in ```exercism/rust``` for example

File: exercises/meetup/example.rkt (hunk 1 above)
Comment (jgilray) [reply exercises/meetup/example.rkt]: I will fix this

**Comment by jgilray** [comment]
OK, looking a little closer, I see this:
```
meetup/meetup.rkt:3:9: collection not found
for module path: math/number-theory
collection: "math"
in collection directories:
/home/travis/.racket/6.6/collects
/home/travis/racket/collects
```
Do we need to edit ```/home/travis/.../collects``` somehow?

**Comment by jgilray** [comment]
OK, I simply modified ```example.rkt``` to remove the dependency on math/number-theory and that seemed to do the trick, @yurrriq.

**Comment by yurrriq** [comment]
LGTM, thanks!

**Pull request merged**